src/app/Dashboard/LayoutTemplatePicker.tsx

```diff
@@ -0,0 +1,578 @@
+      });
+      setIsSortSelectOpen(false);
+    },
+    [setSelectedSort, setIsSortSelectOpen]
+  );
+
+  const sortArrowIcon = React.useMemo(() => {
+    if (!selectedSort) {
+      return <ArrowsAltVIcon />;
+    }
+    return sortDirection === 'asc' ? <LongArrowAltUpIcon /> : <LongArrowAltDownIcon />;
+  }, [selectedSort, sortDirection]);
+
+  const onSortDirectionChange = React.useCallback(() => {
+    setSortDirection((prev) => (prev === 'asc' ? 'desc' : 'asc'));
+  }, [setSortDirection]);
+
+  const onDrawerCloseClick = React.useCallback(() => {
+    setIsDrawerExpanded(false);
+  }, [setIsDrawerExpanded]);
+
+  const panelContent = React.useMemo(() => {
+    return (
+      <DrawerPanelContent isResizable defaultSize="35%">
+        <DrawerHead>
+          {selectedTemplate ? (
+            <DescriptionList isFillColumns>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Name</DescriptionListTerm>
+                <DescriptionListDescription>{selectedTemplate?.name}</DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Description</DescriptionListTerm>
+                <DescriptionListDescription>{selectedTemplate?.description}</DescriptionListDescription>
+              </DescriptionListGroup>
+              {selectedTemplate?.vendor && (
+                <DescriptionListGroup>
+                  <DescriptionListTerm>Vendor</DescriptionListTerm>
+                  <DescriptionListDescription>{selectedTemplate.vendor}</DescriptionListDescription>
+                </DescriptionListGroup>
+              )}
+              {selectedTemplate?.version && (
+                <DescriptionListGroup>
+                  <DescriptionListTerm>Version</DescriptionListTerm>
+                  <DescriptionListDescription>{selectedTemplate.version}</DescriptionListDescription>
+                </DescriptionListGroup>
+              )}
+              <DescriptionListGroup>
+                <DescriptionListTerm>Preview</DescriptionListTerm>
+                {
+                  <div className="dashboard-layout-preview">
+                    <ServiceContext.Provider value={fakeServices}>
+                      <ChartContext.Provider value={fakeChartContext}>
+                        <Grid
+                          id={'dashboard-layout-preview-grid'}
+                          hasGutter
+                          onClick={(e) => {
+                            e.preventDefault();
+                            e.stopPropagation();
+                          }}
+                        >
+                          {selectedTemplate.cards
+                            .slice(0, CARD_PREVIEW_LIMIT)
+                            .filter((cfg) => hasConfigByName(cfg.name))
+                            .map((cfg, idx) => (
+                              <FeatureFlag level={getConfigByName(cfg.name).featureLevel} key={`${idx}-wrapper`}>
```

Comment:
I think the feature flag should apply to the dashboard template? Maybe take the lowest feature level of all cards in the template as the template's level?

This way, there is no need to check feature level here?

Reply:
Also, its not showing empty state for me here...


![Screenshot from 2023-04-11 21-59-22](https://user-images.githubusercontent.com/68053619/231333584-3fb2a433-b4d4-4839-90be-1d1d7cd53dc5.png)

Reply:
Sometimes it does.
![Screenshot from 2023-04-11 22-48-22](https://user-images.githubusercontent.com/68053619/231335312-28b4624c-c249-4b21-94a0-e0dc711bfce1.png)

Reply:
There is still one problem tho. The fake services are meant to work only for a single chart type (i.e. same card type). For example, Mbean Metrics, fake data are just for Process CPU Load.

Maybe we can just instead show a table/data list of included cards?

Reply:
Oh wait actually no. I think u can add more fake Mbean fields in fake services right? This should work just fine for preview.

Reply:
Here, just add more fake fields in.

https://github.com/cryostatio/cryostat-web/blob/833fddc8986a07f14ff5c84f966350bf9763ea13/src/app/utils/fakeData.ts#L173